multitenancy-benchmarks/require-resource-quota/.chainsaw-test/chainsaw-test.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: chainsaw.kyverno.io/v1alpha1
+kind: Test
+metadata:
+  creationTimestamp: null
+  name: require-resource-quota
+spec:
+  steps:
+  - name: step-01
+    try:
+    - apply:
+        file: ns-good.yaml
+    - script:
+        content: |
+          sed 's/validationFailureAction: audit/validationFailureAction: Audit/' ../require-resource-quota.yaml | kubectl create -f -
+    - assert:
+        file: chainsaw-step-01-assert-1.yaml
+  - name: step-02
+    try:
+    - apply:
```

Comment:
@fykaa  If this is expected to produce an error or fail, then the policy's validationFailure action should be set to **Enforce** mode. Could you please update the validationFailureAction comment by changing the value to Enforce mode: `validationFailureAction: Audit/validationFailureAction: Enforce`